docs/organizations/segments.md

```diff
@@ -0,0 +1,37 @@
+# Segments
+Segments are dimensions that Codacy reads from your provider that organizes repositories into relevant groups for you. Today, Segments is available for: [GitHub Custom Properties](#github-custom-properties)
+  
+<br><br>
+### How can Segments be utilised?
+- [Repository list](../managing-repositories/#provider-segments) 
+- [Security & Management Risk](../managing-security-and-risk/)
+
+#### Enabling Segments
+To enable **Segments**, an initial sync between your provider and Codacy needs to happen. Once completed, you can use **Segments** to better locate and organize repositories within Codacy.
+<br>
```

Comment:
This is repeated in the "managing repositories" page 👀

Reply:
Is it? (the preview is not working) I cannot see it in PR 🤔

Reply:
Ins't this paragraph basically explaining the same? https://github.com/codacy/docs/pull/2214/files#diff-5407c16393192ee2f43a31272e969d8f0ac37cac44a89255f858dcd6791870b4R78

Reply:
I wonder if it'd be better to move the more detailed information found in the `Repository list` page to the `Segments` page, and in the `Repository list` have a link back to a ~`Configure segments` part of the `Segments` page.

Move this 👇 to `Segments` page.
![Screenshot 2024-10-09 at 11 34 14](https://github.com/user-attachments/assets/0875b86e-259a-496a-a6fb-6f2070024780)

Replacing by something like this👇 on `Repository list`, maybe even an info box:
```
!!! info "Check out how to [enabled and configure segments](../segments/#enabling-segments)"
```

This info block could also be used in the `Managing security and risk` pages.